sdk/keyvault/Azure.Security.KeyVault.Keys/src/Azure.Security.KeyVault.Keys.csproj

```diff
@@ -3,7 +3,6 @@
   <PropertyGroup>
     <Description>This is the Microsoft Azure Key Vault Keys client library</Description>
     <AssemblyTitle>Microsoft Azure.Security.KeyVault.Keys client library</AssemblyTitle>
-    <Version>4.0.2</Version>
```

Comment:
Any idea how this bad merge happened? Was it incorrectly merged? I would have expected a merge conflict so we would have needed to take both in order to get into this state.

Reply:
Not sure how it got merged that way, but the effect would've been the same since the next `Version` would've overwrote this first one. Could have something to do with .gitattributes using `merge=union` for *.csproj. In most other repos, .csproj are treated as any other text file so a merge conflict would've happened (if not resolved automatically). Can't say I've seen this before without `merge=union`.

Reply:
Thanks for the pointer I think you are correct that is what caused this. After some reading of what union does I don't think we want that for our repo so I'm going to remove it.

Reply:
https://github.com/Azure/azure-sdk-for-net/pull/10440